internal/pkg/app/app_test.go

```diff
@@ -0,0 +1,80 @@
+package app
+
+import (
+	"errors"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/viper"
+	testLog "github.com/xmidt-org/ears/test/log"
+	"go.uber.org/fx"
+	"os"
+	"testing"
+	"time"
+)
+
+func AppConfig() Config {
```

Comment:
https://github.com/spf13/viper#working-with-multiple-vipers

Create separate vipers.  These may end up interfering with each other.

Reply:
I guess you decided against the small change of:
```
	v := viper.New()
	v.Set("ears.logLevel", "info")
	v.Set("ears.api.port", 8080)
	v.Set("ears.env", "test")
	return v.Sub("ears")
```
?

Reply:
I thought that is what I've done